Change the file name and provide English version. (#3)
ByteLegend/game-data: JavaIsland/docs/star-bytelegend/register-github-account-en.md

> # How to create a GitHub account

Reviewer: 英文标题的一般习惯是`How to Create a GitHub Account`。

Reviewer: 也就是除了虚词之外的单词都首字母大写。

> Open [GitHub](https://github.com), click **Sign up** at the upper right corner. You will go to the register page.

Reviewer: `register`是一个非常chinglish的"注册"的说法，正确的说法是`sign up page`

> ![](https://raw.githubusercontent.com/ByteLegend/game-data/master/JavaIsland/docs/star-bytelegend/register-github-account/sign-up-gh.png)

Reviewer: ```suggestion
![sign-up-gh.png](https://raw.githubusercontent.com/ByteLegend/game-data/master/JavaIsland/docs/star-bytelegend/register-github-account/sign-up-gh.png)
```

虽然不是什么大问题，但是有的markdow编辑器一定要你在方括号里放个东西

Enter your **email address**, **password**, and the **username** of GitHub to create a GitHub account.

> ![](https://raw.githubusercontent.com/ByteLegend/game-data/master/JavaIsland/docs/star-bytelegend/register-github-account/type-info.png)

Reviewer: ```suggestion
![type-info.png](https://raw.githubusercontent.com/ByteLegend/game-data/master/JavaIsland/docs/star-bytelegend/register-github-account/type-info.png)
```

> Receive the **verification code** and fill it to GitHub.

Reviewer: ```suggestion
Get the **verification code** and fill it out.
```

![](https://raw.githubusercontent.com/ByteLegend/game-data/master/JavaIsland/docs/star-bytelegend/register-github-account/email.png)

You can skip the personalization. Of course, if you want, you can fill it.

![](https://raw.githubusercontent.com/ByteLegend/game-data/master/JavaIsland/docs/star-bytelegend/register-github-account/skip-personalization.png)

> Now, you successfully create your own GitHub account.

Reviewer: ```suggestion
Now, you've created your own GitHub account successfully.
```
